src/App.js

```diff
@@ -41,6 +51,10 @@ class App extends Component {
   render() {
     return (
       <section className='content'>
+        <Switch>
+          <Route path='/upload'></Route>
+          <Route path='/:id'  children={<h2>{this.props.match.params.id}</h2>}></Route>
```

Comment:
```suggestion
          <Route path='/:id'  children={<ImagePage />}></Route>
```
Aqui será o componente  `ImagePage`, e dentro dele que terá o `<h2>{this.props.match.params.id}</h2>`, DENTRO do componente.

Reply:
Não sei exatamente a diferença do `render` para o `children`, necessário estudo.